package.json

```diff
@@ -72,13 +73,23 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "lint-js": "eslint 'src/**/*.{js,jsx}'",
-    "lint-ts": "tslint -c tslint.json --project ./tsconfig.json --format stylish 'src/**/*.{ts,tsx}'"
+    "lint-ts": "tslint -c tslint.json --project ./tsconfig.json --format stylish 'src/**/*.{ts,tsx}'",
+    "test:coverage": "set CI=true && react-scripts test --coverage",
+    "coveralls": "set CI=true && yarn test --runInBand --verbose --coverage=true --forceExit --detectOpenHandles && cat ./coverage/lcov.info | coveralls"
```

Comment:
@p-netm Nice thought, if you run the command locally, the coveralls API will fail to be called since the repo token is not set. Does setting it make it work locally? Let me check

Reply:
@p-netm Seems like since the coveralls API is only triggered by CI services supported by coveralls. Not so sure though

Reply:
@kelvin-muchir I think we can leave this out of the package.json, rather than go through the headache of trying to fix it. It can be hardocded in the config.yml.